content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,667 @@
+      bar, and under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.
+
+      In the drop-down menu that appears, enter a name for the report. By default, the report is
+      named "Pipeline".
+
+      - Optional: Checking the :guilabel:`Default filter` box sets this report as the default
+        analysis when opening the :guilabel:`Pipeline Analysis` page.
+      - Optional: Checking the :guilabel:`Shared` box makes this report available to other
+        users.
+
+      Finally, click :guilabel:`Save`. The report is now saved under the :guilabel:`Favorites`
+      heading.
+
+      .. image:: win_loss/save-to-favorites.png
+         :align: center
+         :alt: Under the Favorites heading, click Save current search and save the report for later.
+
+   .. tab:: Add to a Spreadsheet
+
+      Inserting a report into a spreadsheet not only saves a copy of the report, it allows the
+      adding of charts and formulas similar to an Excel file.
+
+      To add a report in to a spreadsheet:
+
+      - **If in Graph or Pivot View**: Click either the :guilabel:`Insert in spreadsheet` button,
+        and in the pop-up menu that appears, click :guilabel:`Confirm`.
+      - **If in Cohort or List View**: Click the gear-shaped :guilabel:`Actions` icon, hover over
+        :guilabel:`Spreadsheet`, and click either :guilabel:`Insert/Link in spreadsheet`.
+
+        .. image:: win_loss/pivot-view-in-spreadsheet.png
+           :align: center
+           :alt: Pivot View reports especially benefit from being inserted in spreadsheets.
+
+      .. tip::
+         After modifying a spreadsheet and adding additional formulas, consider then adding the
+         entire spreadsheet to a dashboard by clicking :menuselection:`File --> Add to dashboard`.
+
+         In the resulting pop-up menu, name the spreadsheet and select a :guilabel:`Dashboard
+         Section` to house the report. Once complete, click :guilabel:`Create`.
+
+         Using this method, the spreadsheet can be added to a public dashboard instead of only
+         :guilabel:`My Dashboard`.
+
+   .. tab:: Add to a Dashboard
+
+      Adding a report to a dashboard saves it for later and makes it easy to view alongside the rest
+      of :guilabel:`My Dashboard`.
+
+      To add a report to :guilabel:`My dashboard`:
+
+      At the :guilabel:`Pipeline Analysis` page, click the gear-shaped :guilabel:`Actions` icon, and
+      in the resulting drop-down menu, hover over :guilabel:`Dashboard`.
+
+      Under the :guilabel:`Add to my dashboard` heading, enter a name for the report (by default, it
+      is named "Pipeline") and click :guilabel:`Add`.
+
+      To see the saved report, return to the main apps page, and navigate to
+      :menuselection:`Dashboards app --> My Dashboard`.
+
+      .. image:: win_loss/add-to-dashboard.png
+         :align: center
+         :alt: To access the saved report, open the Dashboard app and click My Dashboard.
```

Comment:
I believe this section can be deleted, as well -- since you've already covered it in great detail previously in the doc.